rabbit-source/src/main/java/org/springframework/cloud/stream/module/rabbit/source/RabbitSourceProperties.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.module.rabbit.source;
+
+import javax.validation.constraints.NotNull;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Properties for the Rabbit Source.
+ *
+ * @author Gary Russell
+ *
+ */
+@ConfigurationProperties
+public class RabbitSourceProperties {
+
+	/**
+	 * Whether rejected messages should be requeued.
+	 */
+	private Boolean requeue;
+
+	/**
+	 * Whether the channel is transacted.
+	 */
+	private Boolean channelTransacted;
+
+	/**
+	 * The queues to which the source will listen for messages.
+	 */
+	private String queues;
+
+	/**
+	 * Headers that will be mapped.
+	 */
+	private String mappedRequestHeaders = "STANDARD_REQUEST_HEADERS";
+
+	/**
+	 * Initial retry interval when retry is enabled.
+	 */
+	private int initialRetryInterval = 1000;
+
+	/**
+	 * Max retry interval when retry is enabled.
+	 */
+	private int maxRetryInterval = 30000;
+
+	/**
+	 * Retry backoff multiplier when retry is enabled.
+	 */
+	private double retryMultiplier = 2.0;
+
+	/**
+	 * The maximum delivery attempts when retry is enabled.
+	 */
+	private int maxAttempts = 3;
```

Comment:
Yeah... I won't mind. But especially this one will make undesired, "unretryable" affect:
```
public boolean canRetry(RetryContext context) {
	Throwable t = context.getLastThrowable();
	return (t == null || retryForException(t)) && context.getRetryCount() < maxAttempts;
}
```
:smile: